contracts/src/v0.8/functions/dev/1_0_0/RouterBase.sol

```diff
@@ -231,15 +210,13 @@ abstract contract RouterBase is IRouterBase, Pausable, ITypeAndVersion, Confirme
     if (block.number < s_proposedContractSet.timelockEndBlock) {
       revert TimelockInEffect();
     }
-    s_minorVersion = s_minorVersion + 1;
-    if (s_patchVersion != 0) s_patchVersion = 0;
     // Iterations will not exceed MAX_PROPOSAL_SET_LENGTH
     for (uint8 i = 0; i < s_proposedContractSet.ids.length; ++i) {
```

Comment:
using uint256 is cheaper as it casts to uint256 anyway and there is no downside of using the entire slot in memory as there is no packing in memory